hack/update-docs

```diff
@@ -3,7 +3,7 @@
 set -eu -o pipefail
 
 : "${BUILDX_CMD=docker buildx}"
-: "${FORMATS=md}"
+: "${FORMATS=md,yaml,man}"
```

Comment:
This would mean we vendor all formats. Totally fine by me, it alleviates a step in updating the CLI reference downstream (we only need to re-vendor and not manually generate and copy over yaml)

Reply:
This is just for testing, I don't want to merge this :innocent:

Reply:
aw I was getting excited :)